<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: #FF5733;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="confFileDetect_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>152</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #818181;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 border-color: red;
min-height:40px;
 min-width:150px;
max-height:40px;
max-width:150px;</string>
    </property>
    <property name="text">
     <string>Detect configuration file</string>
    </property>
   </widget>
   <widget class="QLabel" name="confFileDetectLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Configuration file directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetID_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>152</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #818181;
 border-style: solid;
 border-width:1px;
 border-radius:15px;
 border-color: red;
min-height:50px;
 min-width:150px;
</string>
    </property>
    <property name="text">
     <string>Reset ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="idResetResult_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
